app/models/decision.rb

```diff
@@ -10,6 +10,6 @@ class Decision < ApplicationRecord
     message: "Please select Yes or No" }
 
   def comment_made?
```

Comment:
Please could we include `public` in the method name here to differentiate it from the private comments.